src/components/content/order/scale/Scale.tsx

```diff
@@ -85,6 +94,26 @@ export const Scale = ({
         setModifyStatus(status);
     };
 
+    const onClickScale = () => {
+        if (getModificationImpact) {
```

Comment:
do we need this check? this is a mandatory field. so the property will exist always.